modules/heat_conduction/src/materials/GapConductance.C

```diff
@@ -276,12 +279,33 @@ GapConductance::computeQpConductance()
   }
 }
 
+Real
+GapConductance::gapAttenuation(Real adjusted_length, Real min_gap, unsigned int min_gap_order)
+{
+  mooseAssert(min_gap > 0, "min_gap must be larger than zero.");
+
+  if (adjusted_length > min_gap)
+    return 1.0 / adjusted_length;
+  else
+    switch (min_gap_order)
+    {
+      case 0:
+        return 1.0 / min_gap;
+
+      case 1:
+        return 1.0 / min_gap - (adjusted_length - min_gap) / (min_gap * min_gap);
+
+      default:
+        ::mooseError("Invalid Taylor expansion order");
```

Comment:
why the colons before mooseError?

Reply:
That's the scope resolution operator. The member function does not work in static functions. With the colons I select the global function version of mooseError.

Reply:
Gotcha